cmd/spire-server/cli/entry/util.go

```diff
@@ -1,116 +1,122 @@
 package entry
 
 import (
-	"encoding/json"
```

Comment:
According to [this comment](https://github.com/spiffe/spire/pull/3445#discussion_r978932753) by @MarcosDY, "we use spaces". Should I use spaces or tabs for the codebase?

Reply:
The convention inside of the .proto files is spaces for indentation. I don't know how we landed on that convention but anecdotally there are some formatting tools out there that uses spaces for indentation.
For .go files, the natural choice is tabs, since that is what the gofmt family of tools will automatically reformat the code to use. Our linting requires that files are formatted accordingly.